tasks/main.yml

```diff
@@ -35,3 +38,12 @@
               - "{{ ansible_distribution_release }}-backports"
             uris:
               - http://deb.debian.org/debian
+          register: add_backports_repo
+
+        - name: Update the package cache
+          ansible.builtin.apt:
+            force_apt_get: true
```

Comment:
Why `force_apt_get: true` here?  That shouldn't be necessary.

Reply:
I can remove it but I was just duplicating the logic from [pre-#41](https://github.com/cisagov/ansible-role-backports/blob/f53ca3e0519177989199162ade0f7f56d62336d9/tasks/main.yml#L21-L26). There was [no documentation](https://github.com/cisagov/ansible-role-backports/commit/4d885350b369b9e07bdf122888969b3840bff419) around why the option was used so I left it in place (this was before we had `first-commits` pull requests so there's also no pull request to reference).

Reply:
I would remove it.  In 99% of cases this will change nothing since our [cisagov/ansible-role-upgrade](https://github.com/cisagov/ansible-role-upgrade/blob/29c7dafe9e43604eaea6765b6d4d7b0777cf2433/tasks/apt.yml#L2-L6) role will have already run.  In the remaining 1% of cases the `aptitude` package will get automatically installed.

Reply:
Please see [9861ba6](https://github.com/cisagov/ansible-role-backports/pull/42/commits/9861ba677d40146ca04f9c369a6a236943f81141).